_data/iambonface/iambonface.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Bonface Mudiri # No longer than 28 characters
+institution: University of British Columbia 🚩 # no longer than 58 characters
+profile_pic: iambonface.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: Unless you try, you will never learn new tricks! # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
+github_user: iambonface
+---
+
+
+
```

Comment:
Could you delete the extra lines(from 8-10)

Reply:
Spacing Fixed